libs/hdf-converters/src/neuvector-mapper.ts

```diff
@@ -0,0 +1,311 @@
+  checks?: RESTBenchItem[];
+  secrets?: RESTScanSecret[];
+  setid_perms?: RESTScanSetIdPerm[];
+  envs?: string[];
+  labels?: Record<string, string>;
+  cmds?: string[];
+  signature_data?: RESTScanSignatureInfo;
+};
+
+type RESTScanRepoReport = {
+  verdict?: string;
+  image_id: string;
+  registry: string;
+  repository: string;
+  tag: string;
+  digest: string;
+  size: number /* int64 */;
+  author: string;
+  base_os: string;
+  created_at: string;
+  cvedb_version: string;
+  cvedb_create_time: string;
+  layers: RESTScanLayer[];
+} & RESTScanReport;
+
+type RESTScanLayer = {
+  digest: string;
+  cmds: string;
+  vulnerabilities: RESTVulnerability[];
+  size: number /* int64 */;
+};
+
+export type NeuvectorScanJson = {
+  report: RESTScanRepoReport;
+  error_message: string;
+};
+
+const CWE_NIST_MAPPING = new CweNistMapping();
+
+function cweTags(description: string): string[] | undefined {
+  const regex = /CWE-\d{3}/g;
+  return description?.match(regex)?.flat();
+}
+
+function nistTags(cweTags: string[] | undefined): string[] {
+  const identifiers =
+    cweTags?.map((tag: string) => tag.match(/\d{3}/g)?.[0] ?? [])?.flat() ?? [];
+  return CWE_NIST_MAPPING.nistFilter(
+    identifiers,
+    DEFAULT_UPDATE_REMEDIATION_NIST_TAGS
+  );
+}
+
+function ghsaTag(name: string): string | undefined {
+  const regex = /GHSA-[a-z|0-9]{4}-[a-z|0-9]{4}-[a-z|0-9]{4}/;
+  return regex.exec(name)?.[0];
+}
+
+function rhsaTag(name: string): string | undefined {
+  const regex = /RHSA-[a-z|0-9]{4}:[a-z|0-9]{4}/;
+  return regex.exec(name)?.[0];
+}
+
+function cveIdMatches(cveName: string): (value: RESTModuleCve) => boolean {
+  return (cve: RESTModuleCve) => cve.name === cveName;
+}
+
+function universalTags(): MappedTransform<
+  {[key: string]: any} & ILookupPath,
+  ILookupPath
+> {
+  // Heimdall currently doesn't have a way to display passthrough data, and this information would be useful to view per vulnerability.
+  return {
+    envs: {
+      path: '$.report.envs',
+      transformer: (envs: string[]) => (!envs ? undefined : envs)
+    },
+    cmds: {
+      path: '$.report.cmds',
+      transformer: (cmds: string[]) => (!cmds ? undefined : cmds)
+    }
+  };
+}
+
+export class NeuvectorMapper extends BaseConverter {
+  withRaw: boolean;
+  rawData: NeuvectorScanJson;
+  getModule: (moduleName: string) => RESTScanModule | undefined;
+
+  memoizedGetModule(): (moduleName: string) => RESTScanModule | undefined {
+    return (moduleName: string) => {
+      return this.rawData.report.modules?.find(
+        (value: RESTScanModule) => value.name === moduleName
+      );
+    };
+  }
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion,
+      target_id: null
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+      duration: null
+    },
+    profiles: [
+      {
+        name: 'Neuvector',
```

Comment:
```suggestion
        name: 'NeuVector Scan',
```

Please correct the capitalization of 'NeuVector' to match their preference wherever you use it, especially in user facing areas.  Also please match the patterns that we've set in other mappers, ex. making 'name' equal to 'Tool Scan'.